api/orchestration/v1alpha1/kvcache_types.go

```diff
@@ -69,9 +69,9 @@ type MetadataStorage struct {
 
 type CacheSpec struct {
 	// Replicas is the number of kvcache pods to deploy
-	// +kubebuilder:validation:Required
 	// +kubebuilder:default:=3
-	Replicas int `json:"replicas,omitempty"`
+	// +optional
+	Replicas *int `json:"replicas,omitempty"`
```

Comment:
separate PR

Reply:
This leads to a compile error so I changed here together, a separate PR here https://github.com/vllm-project/aibrix/pull/740